packages/web/src/components/static-image/StaticImage.tsx

```diff
@@ -6,12 +6,11 @@ import { Flex } from '@audius/harmony/src/components/layout/Flex'
 import { developmentConfig } from '@audius/sdk/src/sdk/config/development'
 import { productionConfig } from '@audius/sdk/src/sdk/config/production'
 import { stagingConfig } from '@audius/sdk/src/sdk/config/staging'
-import {
-  AppAuth,
-  getDefaultStorageNodeSelectorConfig
-} from '@audius/sdk/src/sdk/services'
+import { AppAuth } from '@audius/sdk/src/sdk/services/Auth/AppAuth'
 import { DiscoveryNodeSelector } from '@audius/sdk/src/sdk/services/DiscoveryNodeSelector/DiscoveryNodeSelector'
+import { getDefaultDiscoveryNodeSelectorConfig } from '@audius/sdk/src/sdk/services/DiscoveryNodeSelector/getDefaultConfig'
 import { StorageNodeSelector } from '@audius/sdk/src/sdk/services/StorageNodeSelector/StorageNodeSelector'
+import { getDefaultStorageNodeSelectorConfig } from '@audius/sdk/src/sdk/services/StorageNodeSelector/getDefaultConfig'
```

Comment:
import still needed?

Reply:
yes, its used a little below for hte storage node selector